c2rust-transpile/tests/snapshots/snapshots__transpile-linux.snap

```diff
@@ -0,0 +1 @@
+snapshots__transpile-linux@rotate.c.snap
```

Comment:
What generates this?

Reply:
Ah, these are symlinks.  `insta` does that removing common prefix thing here again.  I'd rather just put a dummy `empty.c` file in `snapshots/platform-specific`, and then once we add another real test we can remove it.

Reply:
Done.